src/notify.js

```diff
@@ -28,6 +28,8 @@ function Notify(title, options = {}) {
         notifyError = null,
         closeOnClick = false,
         timeout = null,
+        swUrl = null,
```

Comment:
Ok, feel free to leave this in if it's something you need. I still think this is an edge case though.